megamek/src/megamek/server/commands/ChangeWeatherCommand.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (C) 2024 Luana Scoppio (luana.coppio@gmail.com)
```

Comment:
Are you claiming copyright of this code? Normally we ask that the code be under the MegaMek team copyright, I believe

Reply:
Just confirmed with the team, this isn’t the issue I thought it was, so carry on! :)

Reply:
I kinda copied and pasted what were in other files, since this is based directly in an older file that is the same license I tought that adding my name to it to make clear who authored would be reasonable.
But either way, I will review if I didnt forget the megamek team entry in any of the files.